packages/related-keyphrase-suggestions/src/components/KeyphrasesTable/index.js

```diff
@@ -197,8 +198,8 @@ export const KeyphrasesTable = ( { columnNames = [], data, renderButton, related
 		<Table.Body>
 			{ rows && rows.map( ( rowData, index ) => (
 				<KeyphrasesTableRow
-					key={ `related-keyphrase-${ index }` }
-					id={ `related-keyphrase-${ index }` }
+					key={ `${ suffix }-related-keyphrase-${ index }` }
+					id={ `${ suffix }-related-keyphrase-${ index }` }
```

Comment:
This "suffix" is at the start in RTL 😉 

Probably better to call it `idPrefix` or something?
Noting what it is used for in the name.

Reply:
good point